go.mod

```diff
@@ -116,3 +116,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
+
+replace sigs.k8s.io/karpenter => /Users/schalor/Documents/github.com/rschalo/karpenter
```

Comment:
This needs to be changed as soon as the update to the karpenter core is merged.

Reply:
Updated this to point directly at your fork so we could run E2E. Will still need to revert the change, and replace it with a version bump once the upstream PR is merged.